examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml

```diff
@@ -0,0 +1,44 @@
+<Specimen xmlns='http://hl7.org/fhir'>
+  <id value='Specimen-PheobeSmitham-Example'/>
```

Comment:
Id does not conform to file name as expected in UKCore Design guide.
@ryma2fhir Given that this resource also exist in Genomics IG with id: Specimen-PheobeSmitham-Example. Would you give your opinion on this?

Reply:
I think the best way would be to add it to UK Core as `UKCore-Specimen-PheobeSmitham-Example` and if it's the same example in genomics, could we render the example from UK Core package into the genomics IG?

Reply:
I can see there's already an example with the Id : UKCore-Extension-SampleCategory-Example . And as per the Quality Control checker if i'll give the id value as : UKCore-Extension-SampleCategory-PheobeSmitham-Example seems to be a wrong format. So, should i give the id value as something like UKCore-Extension-SampleCategory-Example2? @ryma2fhir can you give your suggestion?

Reply:
We have done this before, so lets go with `UKCore-Extension-SampleCategory-PheobeSmitham-Example`
https://github.com/NHSDigital/FHIR-R4-UKCORE-STAGING-MAIN/blob/develop/examples/UKCore-Extension-PriorityReason-SendingAsText-Example.xml